## Homework 3

**Kevin Moyung**  
**A12609930**  
**Due 11/8/18**  

### Lecture 7 & 8. RNA-seq Analysis

#### Question 1 (15 pts)

**After a sequencing run, we obtain a raw FASTQ file. Then, we use TopHat for read alignment and Cufflinks for transcript compilation and gene identification.**

**(a) What is the content in a FASTQ file? (5 pts)**

**(b) What is the meaning of Phred quality score = 30? (5 pts)**

**(c) What is the meaning of CIGAR string? (5 pts)**

#### Question 2 (20 pts)

**The data matrix that we are discussing in class is transcripts per million (TPM) metric for gene expression.**

**(a) What two normalizations are applied to raw read count data to get the TPM metric?**

**(b) Why are these normalizations important? (5 pts)**

**(c) How does TPM differ from FPKM? (5 pts)**

#### Question 3 (10 pts)

**(a) What is count uncertainty and count dispersion? (5 pts)**

**(b) What contents are included in the output file of Cuffdiff? (5 pts)**

### Lecture 10. Genome Interaction Techniques II

#### Question 4 (10 pts)

**(a) How much content in the human genome is protein-coding and non-protein coding genes? (5 pts)**

**(b) What is the challenge of studying RNA-RNA interaction? (5 pts)**

#### Question 5 (25 pts)

**(a) What is the main information that MARIO provided? (5 pts)**

**(b) Describe how MARIO works. (10 pts)**

**(c) What is the novelty and advantages of MARIO? (10 pts)**

#### Question 6

**(a) What is the main information that MARGI provided? (5 pts)**

**(b) What are the two conclusions of non-coding RNA interaction studies by using MARGI techniques? (10 pts) Hint: Use the study that covered in class to answer this question.**
